### ISSUE TITLE
craft.act behaviour is broken

### ISSUE THREAD

**Issue by cjpappas** [cjpappas/MacSim#39]
Only works if you start the simulation from the tab - won't work if you refresh page. Need to have a check to determine if a sim is running -> call `craft.act`

**Comment by cjpappas** [comment]
Looks like we will address this by integrating the act function within the `init` method
```js
const init = (...) => {
  ...
  craft = { ..., act, ... }
  setInterval(() => ... act() ..., 1000);
  return craft
}
```

**Comment by cjpappas** [comment]
Fixed in 884fdf2cbcef6b5a39eacd492ecbacfcc1990e08

**Comment by cjpappas** [comment]
craft.act cannot be reassigned for some reason

**Comment by cjpappas** [comment]
Fixed again in 335e19059fc781d0b9ff7676e23d99cb9b331489
